threedi_modelchecker/config.py

```diff
@@ -2435,199 +2290,193 @@ def is_none_or_empty(col):
-        invalid=Query(models.GlobalSetting).filter(
-            models.GlobalSetting.timestep_plus == True,
-            models.GlobalSetting.maximum_sim_time_step == None,
+        column=models.TimeStepSettings.max_time_step,
+        invalid=Query(models.TimeStepSettings).filter(
+            models.TimeStepSettings.use_time_step_stretch == True,
+            models.TimeStepSettings.max_time_step == None,
         ),
-        message="v2_global_settings.maximum_sim_time_step cannot be null when "
-        "v2_global_settings.timestep_plus is True",
+        message="time_step_settings.max_time_step cannot be null when "
+        "time_step_settings.use_time_step_stretch is True",
     ),
 ]
 CHECKS += [
     RangeCheck(
         error_code=1105,
-        column=getattr(models.GlobalSetting, name),
+        column=getattr(models.TimeStepSettings, name),
         min_value=0,
         left_inclusive=False,
     )
     for name in (
-        "sim_time_step",
-        "minimum_sim_time_step",
-        "maximum_sim_time_step",
+        "time_step",
+        "min_time_step",
+        "max_time_step",
         "output_time_step",
     )
 ]
 CHECKS += [
     QueryCheck(
         error_code=1106,
         level=CheckLevel.WARNING,
-        column=models.GlobalSetting.minimum_sim_time_step,
-        invalid=Query(models.GlobalSetting).filter(
-            models.GlobalSetting.minimum_sim_time_step
-            > (0.1 * models.GlobalSetting.sim_time_step)
+        column=models.TimeStepSettings.min_time_step,
+        invalid=Query(models.TimeStepSettings).filter(
+            models.TimeStepSettings.min_time_step
+            > (0.1 * models.TimeStepSettings.time_step)
         ),
-        message="v2_global_settings.minimum_sim_time_step should be at least 10 times smaller than v2_global_settings.sim_time_step",
+        message="time_step_settings.min_time_step should be at least 10 times smaller than time_step_settings.time_step",
     )
 ]
 
 ## 111x - 114x: SIMULATION SETTINGS, numerical
-
 CHECKS += [
     RangeCheck(
         error_code=1110,
         column=models.NumericalSettings.cfl_strictness_factor_1d,
-        filters=models.NumericalSettings.global_settings != None,
         min_value=0,
         left_inclusive=False,
     ),
     RangeCheck(
         error_code=1111,
         column=models.NumericalSettings.cfl_strictness_factor_2d,
-        filters=models.NumericalSettings.global_settings != None,
         min_value=0,
         left_inclusive=False,
     ),
     RangeCheck(
         error_code=1112,
         column=models.NumericalSettings.convergence_eps,
-        filters=models.NumericalSettings.global_settings != None,
         min_value=1e-7,
         max_value=1e-4,
     ),
     RangeCheck(
         error_code=1113,
         column=models.NumericalSettings.convergence_cg,
-        filters=models.NumericalSettings.global_settings != None,
+        # filters=models.NumericalSettings.global_settings != None,
```

Comment:
Why comment it out and not remove it?  Either remove it or explain with an annotation.

Reply:
Oops, these were leftovers from the migration to the upgraded schema. I rather first comment and then remove so it's easier to restore. 2c37bd19786c973d507fd73482b9139bed116323